.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- Please use the search function to check if the bug you found has already been reported by someone else -->
+<!-- If you want to suggest a new feature please use "Feature request" instead -->
```

Comment:
``` <!-- If you want to suggest a new feature please use "Feature request" instead --> ```
``` <!-- If you want to suggest an enhancement to a new feature please use "Enhancement" instead --> ```

Reply:
I guess you wanted to write something like
` <!-- If you want to suggest an enhancement to an existing feature please use "Enhancement" instead -->`?

Reply:
Yes, that was a typo.